modules/platform/nuxeo-platform-task/nuxeo-platform-task-api/src/main/java/org/nuxeo/ecm/platform/task/TaskProvider.java

```diff
@@ -31,7 +30,7 @@
  * @author Laurent Doguin
  * @since 5.5
  */
-public interface TaskProvider extends Serializable {
+public interface TaskProvider {
```

Comment:
I think I had an issue somewhere last time I tried to remove the `Serializable` in these descriptors but I don't remember what... Maybe JSF?

Reply:
I don't see why, although i'll check if all these changes have any impact on JSF.
Maybe a confusion with Seam beans in conversation scope, that need to be Serializable?